sdks/python/apache_beam/io/avroio.py

```diff
@@ -137,6 +139,26 @@ def offset(self):
       return self._offset
 
   def read_records(self, file_name, range_tracker):
+    current_block_offset = -1
+    current_block_size = -1
+
+    def _callback(stop_position):
+      if range_tracker.done():
+        return 0
+
+      # Locking progressing to next block  to provide a consistent result
```

Comment:
We want to make sure that parameters within the callback are for a single block. Updated the comment.